bfd/elflink.c

```diff
@@ -13564,6 +13564,13 @@ _bfd_elf_section_already_linked (bfd *abfd,
 
   flags = sec->flags;
 
+  /* Check if the section type is SHT_COMDAT and the SEC_LINK_ONCE flag is not
```

Comment:
This is mainline code, not sure if you should add this piece of code here. All the backend code should be in arc specific bfd file.